src/styles/blocks/_mojoheader.scss

```diff
@@ -0,0 +1,76 @@
+@import "../utils/variables";
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Helvetica", sans-serif;
```

Comment:
You don't use Helvetica anywhere, and you don't import it in html, so delete it.